packages/terra-clinical-onset-picker/tests/nightwatch/onset-spec.js

```diff
@@ -0,0 +1,150 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Checkout the changes made to the nightwatch tests in this commit: https://github.com/cerner/terra-clinical/commit/71deb3df81d73b159f325e4445ca0b494c6b765b
You'll need to update these tests for them to work with terra-toolkit 2.0